cmds/core/mv/mv.go

```diff
@@ -64,26 +64,26 @@ func moveFile(source string, dest string) error {
 	return nil
 }
 
-func mv(files []string, todir bool) error {
+func mv(update, noClobber bool, files []string, todir bool) error {
```

Comment:
similarly, could this be
mv(files []string, update, noClobber, todir bool) error

Reply:
Sure, updated.